firefox-ios/firefox-ios-tests/Tests/ClientTests/Utils/StoreTestUtility.swift

```diff
@@ -0,0 +1,34 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import Foundation
+import Redux
+@testable import Client
+
+protocol StoreTestUtility {
+    func setupAppState() -> AppState
+    func setupTestingStore()
+    func resetTestingStore()
+}
+
+// Utility class used when replacing the global store for testing purposes
```

Comment:
Nit: i'd add a third ```/``` so it is also visible for documentation

Reply:
updated! will do this going forward and use `///`

any cases where you'll use `//` over `///`? Do you think this would be a good thing to standardize?